ravenframework/Models/EnsembleModel.py

```diff
@@ -246,6 +250,7 @@ def initialize(self,runInfo,inputs,initDict=None):
     for modelClass, modelType, modelName, modelInstance in self.assemblerDict['Model']:
       if not isThereACode:
         isThereACode = modelType == 'Code'
+
```

Comment:
I would suggest we add these lines in Logical model and Hybrid model.

Reply:
I addressed this as well.